ci/base_cuda12.Dockerfile

```
@@ -0,0 +1,57 @@
+FROM docker.io/nvidia/cuda:12.5.0-devel-ubuntu20.04
```

maybe with something like
```suggestion
ARG CUDA_VERSION=12.5.0
FROM docker.io/nvidia/cuda:$CUDA_VERSION-devel-ubuntu20.04
```
we can avoid the duplication

---

Good idea, thanks